src/AWS/CRT/Auth/Signing.php

```diff
@@ -9,6 +9,10 @@
 
 abstract class Signing extends NativeResource {
     static function signRequestAws($signable, $signing_config, $on_complete) {
-        return self::$crt->sign_request_aws($signable, $signing_config, $on_complete);
+        return self::$crt->sign_request_aws($signable->native, $signing_config->native,
+            function($result, $error_code) use ($on_complete) {
+            $signing_result = SigningResult::fromNative($result);
```

Comment:
nitpick: should be tabbed with the line below so it's clearer what the function contains